lib/livebook/notebook/explore/intro_to_livebook.livemd

```diff
@@ -1,20 +1,8 @@
 # Welcome to Livebook
 
-## Introduction
```

Comment:
I removed introduction because we can assume that people who reached this point people know what Livebook is about (given livebook.dev).